docs/pages/guides/receive-output.md

```diff
@@ -0,0 +1,90 @@
+# How to receive output streams
+
+Live Compositor currently supports only RTP (H264 + Opus) as an output format:
+
+### RTP over TCP vs UDP
+
+RTP streams can be delivered over TCP or UDP. Depending on your use case different choice might make more sense, but in general we recommend using TCP if possible.
+
+What to choose?
+- If you are using Membrane Framework plugin all communication already happens over TCP. Currently, we do not support any way to configure it.
+- Some of the popular multimedia tools do not support RTP over TCP e.g. FFmpeg.
+- UDP should only be used for communication on localhost. We do not support retransmission or packet reordering, so if you use it in unreliable network it might lead to unexpected behavior.
```

Comment:
```suggestion
- UDP should only be used for communication on localhost. We do not support retransmission or packet reordering, so if you use it in an unreliable network it might lead to unexpected behavior.
```